orchestrator/orchestrator.go

```diff
@@ -156,24 +156,27 @@ func (s *Orchestrator) processMap(mapping *process.Process_Node_Map, wf *process
 		Fields: make(map[string]*types.Value),
 	}
 	for _, output := range mapping.Outputs {
-		ref := output.GetRef()
-		if ref == nil {
-			continue
-		}
-
-		node, err := wf.FindNode(ref.NodeKey)
-		if err != nil {
-			return nil, err
-		}
+		if ref := output.GetRef(); ref != nil {
+			if ref == nil {
```

Comment:
ref will never be nil here (look above if check)

Reply:
good catch, got some issue with a conflict while rebasing maybe comming from that.